docs/RegistrationChecklist.md

```diff
@@ -0,0 +1,58 @@
+2. Ontology maintainers to ensure their ontology is eligible for admission prior to making a request
+
+The checklist is expected to evolve over time; anyone can contribute to the discussion [here](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1116).
+
+## Checklist
+
+- Principle 1: [Open](http://obofoundry.org/principles/fp-001-open.html)
+  - The ontology MUST be conformant with this principle
+- Principle 2: [Common Format](http://obofoundry.org/principles/fp-002-format.html)
+  - The ontology MUST be conformant with this principle
+  - The ontology MUST be parseable in Protégé (or OWLAPI) and be logically consistent with a standard OWL reasoner such as ELK or HermiT
+- Principle 3: [URI/Identifier Space](http://obofoundry.org/principles/fp-003-uris.html) 
+  - The ontology MUST be conformant with this principle, with the caveat the prefix is not yet registered
+  - The requested namespace MUST be available (not already used by another ontology)
+- Principle 4: [Versioning](http://obofoundry.org/principles/fp-004-versioning.html) 
+  - The ontology MUST have a versionIRI
+- Principle 5: [Scope](http://obofoundry.org/principles/fp-005-delineated-content.html) 
+  - IF there is significant conceptual overlap or matching labels THEN there MUST be computable linkages (one of: skos, obo:xref, owl) which are reasonably comprehensive
+  - There SHOULD be a written plan (e.g ticket) for coordinating with overlapping ontologies
+- Principle 6: [Textual Definitions](http://obofoundry.org/principles/fp-006-textual-definitions.html) 
+  - The ontology SHOULD be conformant with this principle
+  - All ontology root terms MUST have definitions
+  - All ontology terms SHOULD have a definition
+  - Coverage:
+    - Alt 1: At least 20% of terms in the ontology MUST have definitions OR
+    - Alt 2: There should be a very good (explicit reason) for not having definitions on at least 20% of the terms
+  - There SHOULD be a plan for defining all terms
+  - classes SHOULD have defining logical axioms where appropriate
+- Principle 7: [Relations](http://obofoundry.org/principles/fp-007-relations.html)
+  - The ontology SHOULD be conformant with this principle
+- Principle 8: [Documentation](http://obofoundry.org/principles/fp-008-documented.html)
+  - The ontology SHOULD be conformant with this principle
+- Principle 9: [Documented Plurality of Users](http://obofoundry.org/principles/fp-009-users.html)
+  - The submitter SHOULD state existing users and assumed users
+- Principle 10: [Commitment To Collaboration](http://obofoundry.org/principles/fp-010-collaboration.html)
+  - The submitter SHOULD be committed to collaboration
+- Principle 11: [Locus of Authority](http://obofoundry.org/principles/fp-011-locus-of-authority.html)
+  - The ontology MUST be conformant with this principle
+- Principle 12: [Naming Conventions](http://obofoundry.org/principles/fp-012-naming-conventions.html)
+  - All classes MUST have labels
```

Comment:
What about obsolete ones?

Reply:
Maybe we need to add text something like, "To be added to the OBO Foundry, ontologies must meet these requirements. Ontologies that are already listed that do not meet these requirements will be moved to 'inactive' or 'obsolete' status."

Reply:
I meant more, what about obsolete classes; do they have to have labels? I would probably not go into too much depth around when to obsolete an ontology right in this checklist, but we can raise a separate issue!

Reply:
Ah, sorry, I misinterpreted.

Reply:
This should read: all non-obsolete classes must have labels